### ISSUE TITLE
Add contextual usage blurb for new attention-bell icon

### ISSUE THREAD

**Issue by evwilkin** [patternfly/patternfly-org#1980]
Text needed for [icons table](https://www.patternfly.org/v4/design-guidelines/styles/icons) for new attention-bell icon being added for NotificationBadge (see https://github.com/patternfly/patternfly/issues/2994)

**Comment by mceledonia** [comment]
The attention-bell icon is only to be used within our notification badge component, which itself should only be used within our masthead component. It has a specific usage as it only appears when the "attention state" of the notification badge is triggered. It exists to provide a shape based visual indication in addition to the red color switch of the notification badge's attention state.

**Comment by evwilkin** [comment]
@bdellasc this icon has been added in https://github.com/patternfly/patternfly/pull/3309. This icon will go live with our release next week, will you have a chance to review the icon description above and condense it into a blurb for the [contextual usage column on the icons table](https://www.patternfly.org/v4/design-guidelines/styles/icons#all-icons) this week?

**Comment by bdellasc** [comment]
@evwilkin sure. I have some additional feedback for the existing bell icon in the table that also relates to the other status icons. I'll post it here and then we can figure out if it should be a separate issue.

**Comment by bdellasc** [comment]
@mceledonia original description: "The attention-bell icon is only to be used within our notification badge component, which itself should only be used within our masthead component. It has a specific usage as it only appears when the "attention state" of the notification badge is triggered. It exists to provide a shape-based visual indication in addition to the red color switch of the notification badge's attention state.

Might want to name it "bell-attention" so it stays with the basic bell in a list.

@evwilkin - my suggestion for the two columns for bell-attention:
**Type		| Contextual usage**	
Status 		| Represents status: attention. Exclusively used in the 
                          notification badge component within the masthead. 

**Icon table feedback**
All other status icons:
- I think we should show status icons in the color they typically appear (see also warning, ok, info, bell-attention, etc). We did this in PF 3. I think it still is valid in PF 4?
- Existing bell icon: I think a teal bell should be a second icon for the second line in the contextual usage column (see how I did it in my spreadsheet). Force a return for that second line: "Represents status: default notification" so it aligns with the second bell icon in the bell row?
- Regarding the new bell-attention icon, it feels like it should be contained in the same row as the other bell icons as it's another status type for the basic bell? We'd have three in one row: bell, default status bell (teal), bell-attention (red)? Each description would have a forced return to keep them aligned with their icons?

In Documentation:
In the notification drawer pattern ([https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/notification-drawer](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/notification-drawer)), we don't call out the name of the "Notification badge" component [https://www.patternfly.org/v4/documentation/core/components/notificationbadge#component-title]((https://www.patternfly.org/v4/documentation/core/components/notificationbadge#component-title)) in the header bar. This could be a good way to cross-link/name that component between the design guidelines and documentation.

These additional things I found as I was looking at the various pieces when condensing Michael's description. Not sure if some of it belongs in a separate issue.
